client/src/components/InboxUserProfilesGroup/InboxUserProfilesGroup.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import InboxUserProfilesGroup from './InboxUserProfilesGroup';
+
+storiesOf('InboxUserProfilesGroup', module)
```

Comment:
Be sure to update this so this component goes into a sub directory

Reply:
InboxUserProfilesGroup should have been deleted...